src/mixins/common-mixin.ts

```diff
@@ -86,4 +86,71 @@ export default class CommonMixin extends Vue {
     const result = charSet.charAt(intVal)
     return result
   }
+
+  /**
```

Comment:
This was copied over from Edit UI.

Ultimately (but please not in this PR), we could move this mixin to the shared repo.